templates/security/esm.html

```diff
@@ -8,318 +8,560 @@
+          <tbody>
+            <tr>
+              <td>Over 2,300 packages in Ubuntu Main repository</td>
+              <td class="u-align--left">5 years </td>
+              <td class="u-align--left">10 years</td>
+              <td class="u-align--left">10 years</td>
+            </tr>
+            <tr>
+              <td>Over 23,000 packages in Ubuntu Universe repository</td>
+              <td class="u-align--left">Best effort</td>
+              <td class="u-align--left">Best effort</td>
+              <td class="u-align--left">10 years</td>
+            </tr>
+          </tbody>
+        </table>
       </div>
     </div>
+
+    <div class="u-fixed-width">
+      <table aria-hidden="true" class="p-table--esm-chart">
+        <thead class="u-hidden">
+          <th width="6%"></th>
+          <th colspan="2" width="25%">&nbsp;</th>
+          <th colspan="10" width="65%">&nbsp;</th>
+        </thead>
+        <tbody>
+          <tr height="254px">
+            <td class="u-align--right u-table-border--top u-no-padding-right" style="padding-top: 0"><strong>2,300</strong><br /><strong>23,000</strong></td>
+            <td colspan="2" class="u-table-border--top" style="padding-top: 0" width="25.75%"> Ubuntu Main packages, plus<br /> Ubuntu Universe packages</td>
+            <td colspan="10" style="background-color:#000; color:#ffffff;padding-top: 0"><strong>Ubuntu Pro</strong></td>
+          </tr>
+          <tr style="border-top: none; line-height: 8px;">
+            <td class="u-align--right u-table-border--top u-no-padding-right"><strong>2,300</strong></td>
+            <td colspan="2" class="u-table-border--top" style="overflow: hidden;white-space: nowrap;text-overflow: ellipsis;"> Ubuntu Main packages</td>
+            <td colspan="5" style="background-color:#E95420; color:#ffffff; border-top: 1px solid #E95420;"><strong>Ubuntu LTS</strong></td>
+            <td colspan="5" style="background-color:#000; color:#ffffff; border-top:1px solid #000;">&nbsp;</td>
+          </tr>
+          <tr class="u-hide--small u-hide--medium" style="border-top: 0;">
+            <td colspan="2" width="50%">&nbsp;</td>
+            <td class="u-align--right u-table-border--right">GA</td>
+            <td class="u-align--right u-table-border--right">1 year</td>
+            <td class="u-align--right u-table-border--right">2 years</td>
+            <td class="u-align--right u-table-border--right">3 years</td>
+            <td class="u-align--right u-table-border--right">4 years</td>
+            <td class="u-align--right u-table-border--right">5 years</td>
+            <td class="u-align--right u-table-border--right">6 years</td>
+            <td class="u-align--right u-table-border--right">7 years</td>
+            <td class="u-align--right u-table-border--right">8 years</td>
+            <td class="u-align--right u-table-border--right">9 years</td>
+            <td class="u-align--right u-table-border--right" style="padding-left: 0.3rem">10 years</td>
```

Comment:
why padding-left? why not right-align on all cells?

Reply:
It's already aligning to the right `u-align--right`? Any other change would just misalign the table with the top section or the bottom sectin of study cases.